src/App.java

```diff
@@ -13,22 +13,111 @@ public void start() {
         printGreeting(userName);
         printInstructions();
         doMadlib();
-        System.out.println("hi");
+
     }
     
     /* Here you should create all missing methods used by the start() method
      * I have created the signature for ONE of the methods below.
      * You can create any other methods you need as well!
      */ 
-    
+   
+    public String getUserName() {
+       Scanner scan = new Scanner(System.in); 
+        System.out.println("Please enter your name");
+        String name = scan.nextLine();
+        return name;
+    }
+
+    public void printGreeting(String userName) {
+      
+        System.out.println("Hello "+ userName + "! Welcome to the Mad Libs game.");
+       
+    }
+     /** Prints instructions */
     public void printInstructions() {
-        
+
+
+        System.out.println("In this game, you will be asked to provide a series of words");
+        System.out.println("once you've provided the words, fun story will come out with your words!");
+        System.out.println("Now let's get started!");
+       
     }
     
-  
+    public void doMadlib() {
+        String mode;
+        Scanner scan = new Scanner(System.in); 
+        System.out.println("Choose a mode")
```

Comment:
This set of lines of code do not work - it seems you were heading towards an option for multiple madlibs and ran out of time or failed to submit? Is there more I should know about what happened here?